CSS_Components/rating.css

```diff
@@ -0,0 +1,14 @@
+@import '/CSS_Components/card.css';
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro&display=swap');
```

Comment:
why you used this fonts import multiple time..
you can import that in main file right!

Reply:
> why you used this fonts import multiple time.. you can import that in main file right!

Yes yes will do that.